generator/src/Pos/Generator/Block/Logic.hs

```diff
@@ -80,7 +80,7 @@ genBlocks params inj = do
         let finishEOS = toEnum $ fromEnum tipEOS + fromIntegral numberOfBlocks
         foldM' genOneBlock mempty [startEOS .. finishEOS]
 
-    genOneBlock t eos = ((t <>) . inj) <$> genBlock eos
+    genOneBlock t eos = ((t `mappend`) . inj) <$> genBlock eos
```

Comment:
Why `<>` is not used here? Is `t` some type from library for which we can't implement `Semigroup` instance?